### PR TITLE
Getting more updates done on the Link-State 2015 page

### PR DESCRIPTION
**Pull request by thomasrussellmurphy**
This updates the introduction. Things still to happen include:
  * [x] Using our newer logo that is on our sponsorship tiers document
  * [x] Updating any more text to provide relevant information
  * [x] Creating a Facebook event and providing the link

Things that should happen soon:
  * [not yet] Create an EventBrite event, but we will need to discuss ticketing logistics before opening the event
  * [not yet] Generating the appropriate EventBrite iFrame and embedding it
  * [hoping for in the future] Get suggestions from other people on what should be included as we move forward

### CODE REVIEW

**Comment by StephHippo** [comment]
:+1:

**Comment by thomasrussellmurphy** [comment]
One list item down, at least two to go.

**Comment by thomasrussellmurphy** [comment]
Goal: get this PR completed, merged, and deployed before bed.
  - [ ] Success

**Comment by StephHippo** [comment]
Don't create the PR until you're ready to merge. Or break this up into smaller PRs and merge them as you go. Every time I get an email I think you're waiting on someone to check for you to merge.

**Comment by thomasrussellmurphy** [comment]
Alright, moving this to the status of requesting-pull. I just looked over the changes and I think this works well enough. First post updated to reflect current status.

File: Gemfile.lock [hunk 1]
```diff
@@ -91,4 +91,4 @@ DEPENDENCIES
   unicorn
 
 BUNDLED WITH
-   1.10.4
+   1.10.3
```
Comment (ajm188) [inline Gemfile.lock]: Why did bundler version go down?
Also is there a way to remove this? Bundler version doesn't really matter so much as the versions of the gems it's bundling :package: :stuck_out_tongue_winking_eye:

**Comment by ajm188** [comment]
@thomasrussellmurphy do you think that we should create issues for the "things that should happen soon" so they don't get lost once this is merged?

**Comment by thomasrussellmurphy** [comment]
Working on going gung-ho with updating bundle and then `bundle update`.

@ajm188 I'm uncertain about using the issues. They're pretty reasonable neglected so far. GitHub issues will be part of our upcoming discussion of whole-group communication.

**Comment by thomasrussellmurphy** [comment]
Now I've updated gems and tested site operation. Everything looks to still be working.

**Comment by ajm188** [comment]
That's reasonable. Personally, I'm a big fan of using issues, if nothing else as a public reminder to myself of things I want to do. Maybe if we start using issues more fervently, others will follow suit?

**Comment by ajm188** [comment]
alright looks good :ship: :it:

**Comment by thomasrussellmurphy** [comment]
Note made in my agenda re: critical mass/attention.

Shipping now and going to deploy, updating gems along  the way.

**Comment by thomasrussellmurphy** [comment]
Update and deploy successful. Thanks, all!

**Pull request merged**